extensions/amp-lightbox/validator-amp-lightbox.protoascii

```diff
@@ -47,6 +47,12 @@ tags: {  # <amp-lightbox>
   attrs: { name: "controls" }
   attrs: { name: "from" }
   attrs: { name: "scrollable" }
+  attrs: {
+    name: "animate-in"
```

Comment:
please alphabetize

also, repeated fields for value and value_casei is not on github yet, but we'll try to get it there today.

Reply:
Alphabetized.

Waiting for repeated `value_casei` support to make it into the repo and then will merge.